### PR TITLE
Clamp `LinearlyTaperedRestoringMask` to lie within (0, 1)

### PR DESCRIPTION
**Pull request by simone-silvestri**

### CODE REVIEW

**Comment by navidcy** [comment]
What does this do? Is it related to #237?

**Comment by simone-silvestri** [comment]
> What does this do? Is it related to #237?

I don't think so. This is because the mask might be larger than one for particular input choices. 
For example, if the domain spans (-80, 80) in latitude and someone would have specified a linearly tapered polar mask with 
```julia
mask = LinearlyTaperedPolarMake(southern = (-70, -60), northern = (60, 70))
```
between (-80, -70) and (70, 80) the mask would have been larger than one.

**Comment by glwagner** [comment]
Heh, I did not understand that from the PR title. I changed it. Hopefully it's right.

**Comment by glwagner** [comment]
Also this description should be put into the source code.

File: src/DataWrangling/ECCO/ECCO_mask.jl [hunk 1]
```diff
@@ -88,17 +88,18 @@ end
                                southern = (-75, -70),
                                z = (-20, 0))
 
-Build a mask that is linearly tapered in latitude inbetween the northern and southern edges.
-The mask is constant in depth between the z and is equal to zero everywhere else.
+Build a mask that is linearly tapered in latitude between the northern and southern edges.
+The mask is constant in depth between the z and equals zero everywhere else.
+The mask is capped to one if the latitude is beyond the outer northern or southern edges.
```
Comment (glwagner) [inline src/DataWrangling/ECCO/ECCO_mask.jl:93]: Usually we call this "clipping", not "capping".

Comment (glwagner) [reply src/DataWrangling/ECCO/ECCO_mask.jl:93]:
```suggestion
The mask is limited to lie between (0, 1).
```

**Review by glwagner — APPROVED** [review]

**Pull request merged**